src/main/java/madstax/view/PlannerCellRenderer.java

```diff
@@ -26,9 +26,30 @@ public Component getTableCellRendererComponent(JTable table, Object value, boole
             default:
                 s = value.toString();
         }
+
         Component c = renderer.getTableCellRendererComponent(table, s,
                 isSelected, hasFocus, row, col);
-        ((JLabel) c).setHorizontalAlignment(SwingConstants.LEFT);
+        ((JLabel) c).setHorizontalAlignment(SwingConstants.CENTER);
+
+        if(col == 4){
+            switch(s){
+                case "Approved":
+                    c.setForeground(new Color(15, 194,0));
```

Comment:
just curious why some colours are pre-coded .red stuff and others are RGB values? For my own personal learning hahaha

Reply:
Java swing has some built in colours but they don't have the whole range which is why I used RGB in some cases, e.g. their orange is quite bright and doesn't look great in our app so I got the RGB for a darker one